README.md

```diff
@@ -73,15 +76,14 @@ After you're finished please remove all the comments and instructions!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
-
+## 🚀 Live Demo <a name="live-demo"></a>
 
 - [Live Demo Link](https://google.com)
 
```

Comment:
Please kindly note that deployment is not a requirement in this exercise. you can Mention that the ` Live Demo Link is not available` instead of putting Google URL. I see that the live demo link navigates to `https://google.com` 😀and that isn't your portfolio website.